extensions/datetime-range/README.md

```diff
@@ -11,12 +13,12 @@ An extension to provide datetime ranges with a start and an end datetime stamp i
 
 | Field Name         | Type   | Description                                                  |
 | ------------------ | ------ | ------------------------------------------------------------ |
-| dtr:start_datetime | string | **REQUIRED.** The first or start date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
-| dtr:end_datetime   | string | **REQUIRED.** The last or end date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+| start_datetime | string | **REQUIRED.** The first or start date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+| end_datetime   | string | **REQUIRED.** The last or end date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
 
 **Note:** All STAC Items that use this extension MUST include both fields, which enables a user to
 search STAC records by the provided times.
 
 ## Implementations
 
-None yet, still in proposal stage.
+3 proprietary implementations
```

Comment:
Do we have a public one to link to?

Reply:
Not that i'm aware of. Sentinel 1 STAC supposedly uses it, but its not available as a public catalog yet